bitnami/joomla/Chart.yaml

```
@@ -1,22 +1,30 @@
-apiVersion: v1
-name: joomla
-version: 8.1.10
+annotations:
+  category: CMS
+apiVersion: v2
 appVersion: 3.9.22
+dependencies:
+- condition: mariadb.enabled
+  name: mariadb
+  repository: https://charts.bitnami.com/bitnami
+  version: 8.x.x
```

Here we should use the new mariadb major
```suggestion
  version: 9.x.x
```